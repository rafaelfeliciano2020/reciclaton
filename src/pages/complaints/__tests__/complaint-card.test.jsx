import React from 'react';
import renderer from 'react-test-renderer';
import Complaint from '../complaint-card';

it("render styledPerfilDiv", () => {
  const tree = renderer
    .create(
      <Complaint
        imgUrl="https://picsum.photos/200"
        complaintMsg="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque venenatis dolor arcu, id aliquam enim bibendum euismod. Maecenas tempor diam justo, non lobortis purus vulputate vitae. Vestibulum rutrum scelerisque congue. Fusce vestibulum id augue et posuere. Pellentesque a ex semper, sagittis magna ac, sollicitudin sapien."
        reviews={3}
        concordoClick={() => { }}
        discordoClick={() => { }}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
})