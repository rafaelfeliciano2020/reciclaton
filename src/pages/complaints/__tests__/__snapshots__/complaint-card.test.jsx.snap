// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render styledPerfilDiv 1`] = `
<div
  className="sc-bdnylx gkeIqc"
>
  <img
    className="sc-gtssRu kINvcA"
    src="https://picsum.photos/200"
  />
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque venenatis dolor arcu, id aliquam enim bibendum euismod. Maecenas tempor diam justo, non lobortis purus vulputate vitae. Vestibulum rutrum scelerisque congue. Fusce vestibulum id augue et posuere. Pellentesque a ex semper, sagittis magna ac, sollicitudin sapien.
  </p>
  <p>
    3
     Reviews
  </p>
  <div
    className="sc-dlnjPT glnNnG"
  >
    <button
      className="sc-hKFyIo gwzSPB"
      onClick={[Function]}
    >
      Concordo
    </button>
    <button
      className="sc-hKFyIo sc-eCApGN gwzSPB dksaTh"
      onClick={[Function]}
    >
      Discordo
    </button>
  </div>
</div>
`;
